src/styles/blocks/snake.scss

```diff
@@ -0,0 +1,25 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(3, minmax($block-height, 1fr));
```

Comment:
Good job! I totally forgot that we can use "the most powerful grid") I should've used that, too )

Reply:
Thanks for the review 👍